crates/enarx-config/src/lib.rs

```diff
@@ -32,90 +34,26 @@ pub const CONFIG_TEMPLATE: &str = r#"## Configuration for a WASI application in
 # VAR1 = "var1"
 # VAR2 = "var2"
 
-## Pre-opened file descriptors
-[[files]]
-kind = "stdin"
-
-[[files]]
-kind = "stdout"
-
-[[files]]
-kind = "stderr"
-
-## A listen socket
-# [[files]]
-# name = "listen"
-# kind = "listen"
-# prot = "tls" # or prot = "tcp"
-# port = 12345
-
-## An outgoing connected socket
-# [[files]]
-# name = "stream"
-# kind = "connect"
-# prot = "tls" # or prot = "tcp"
-# host = "localhost"
-# port = 23456
-"#;
+# Standard input file
+[stdin]
+kind = "host" # or kind = "null"
 
-const fn default_tcp_port() -> u16 {
-    80
-}
+# Standard output file
+[stdout]
+kind = "host" # or kind = "null"
 
-const fn default_tls_port() -> u16 {
-    443
-}
+# Standard error file
+[stderr]
+kind = "host" # or kind = "null"
 
-fn default_addr() -> String {
-    "::".into()
-}
+## A listen socket on port 12345
+#[listen.12345]
+#prot = "tls" # or prot = "tcp"
 
-#[derive(Clone, Debug, PartialEq, Eq, Serialize)]
-/// Name assigned to a file descriptor
-///
-/// This is used to export the `FD_NAMES` environment variable,
-/// which is a concatenation of all file descriptors names seperated by `:`.
-///
-/// See the [crate] documentation for examples.
-pub struct FileName(String);
-
-impl TryFrom<String> for FileName {
-    type Error = &'static str;
-
-    fn try_from(name: String) -> Result<Self, Self::Error> {
-        if name.find(':').is_some() {
-            Err("file name must not contain ':'")
```

Comment:
Yes

Reply:
Since, as mentioned above, the reliance on the drawbridge_type crate is temporary, a test would be nice to ensure that we continue to reject this even after we replace TreeName with whatever will eventually replace it.

Reply:
reworked